examples/README.md

```diff
@@ -0,0 +1,7 @@
+# Optimizely Go SDK Examples
+
+### Profiling
+Prerequisite: `go get github.com/pkg/profile`
+
+* CPU profile. Execute: `go build -ldflags "-X main.RunCPUProfile=true" main_profile_feature.go && ./main_profile_feature`. It will create cpu.pprof file in your current directory. Then run: `go tool pprof -http=:8080 cpu.pprof` and profile cpu usage using web browser.
```

Comment:
should we add a script for this?

Reply:
for one line, and just an example, I don't see a need.